composer.json

```diff
@@ -24,10 +24,10 @@
     "php": "^7.4 || ^8.0",
     "gedmo/doctrine-extensions": "^3",
     "friendsofsymfony/jsrouting-bundle": "^2",
-    "tijsverkoyen/css-to-inline-styles": "^2.2",
```

Comment:
Waarom is die opeens weg?

Reply:
Is dependency van `"twig/cssinliner-extra": "^3.3",` dus we krijgen die via-via binnen. We gebruiken hem ook niet meer rechtstreeks, maar via die Twig extensie.